projects/components/src/utils/rest/rest-query-search.client.ts

```diff
@@ -65,7 +66,8 @@ export class RestQueryService extends VcdApiClient {
     queryEntity<T extends string>(
         type: T,
         query: RestQuery,
-        options?: RestQueryOptions
+        options?: RestQueryOptions,
+        format?: Format
```

Comment:
Can you make `format` a part of `RestQueryOptions` instead of it's own parameter in the method?